main/Card/index.tsx

```diff
@@ -123,16 +130,16 @@ const Card: FC<Props> = (props) => {
   return (
     <Container style={[outlined ? styles.border : shadowStyle, containerStyle]}>
       {image && <StlyedImage source={image} style={[imageStyle]} />}
-      {renderTitle && (
-        <TitleContainer style={[titleContainerStyle]} hasSubTitle={!!subTitle}>
-          <TitleText style={[titleStyle]}> {title} </TitleText>
-          {subTitle && (
-            <SubTitleText style={[subTitleStyle]}> {subTitle} </SubTitleText>
-          )}
-        </TitleContainer>
-      )}
+
       {children && (
         <ContentsContainer style={[contentsStyle]}>
+          {renderTitle && <TitleContainer style={[titleContainerStyle]} titleTextVertical={titleVertical} hasSubTitle={!!subTitle} >
```

Comment:
All style properties are processing an array, but there is no other style available in the array.
This part is likely to be confusing to other people.
How about passing variables instead of arrays?